homeassistant/components/emoncms/sensor.py

```diff
@@ -221,65 +210,10 @@ def _update_attributes(self, elem: dict[str, Any]) -> None:
         elif elem["value"] is not None:
             self._attr_native_value = round(float(elem["value"]), DECIMALS)
 
-    def update(self) -> None:
+    @Throttle(MIN_TIME_BETWEEN_UPDATES)
```

Comment:
I think the throttle can be removed, because this function isn't called more often, as was the case with the shared update object in the old code

Reply:
Done